packages/ifpack2/src/Ifpack2_Parameters.cpp

```diff
@@ -170,6 +170,7 @@ void getValidParameters(Teuchos::ParameterList& params)
   // params.get("relaxation: damping factor", 1.0); // already set
   // params.get("relaxation: zero starting solution", true); // already set
   params.set("partitioner: type", "greedy");
+  params.set("zoltan2: algorithm", "phg");
```

Comment:
Is `phg` a reasonable default?  (Is it a hypergraph partitioner?)

Reply:
phg is a hypergraph partitioner. I think it's reasonable. Metis would be faster but it's a TPL. Jet is a new partitioner that might be better but not in Zoltan2 yet (coming soon).